doc/dev_guide/internals/box_protocol.rst

```diff
@@ -1073,6 +1075,37 @@ The body is a 2-item map:
 ``IPROTO_EVENT_DATA`` (code 0x57) contains data sent to a remote watcher.
 The parameter is optional, the default value is ``nil``.
 
+..  _box-protocol-shutdown:
+
+Graceful shutdown protocol
+--------------------------
+
```

Comment:
I think that the user deserves an explanation: what is a graceful shutdown, what does it do, why is it a good thing? Perhaps that's what @locker meant.

Consider inserting a brief explanation into the description in the system events doc, too.